autosklearn/util/progress_bar.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from threading import Thread
+from tqdm import trange
+
+
+class ProgressBar(Thread):
+    """A Thread that displays a tqdm progress bar in the console."""
+
+    def __init__(self, total: float, update_interval: float = 1.0, disable: bool = False):
+        """
+        Parameters
+        ----------
+        total: the total amount that the progress bar should reach
+        update_interval: reduce this to update the progress bar more frequently
+        disable: flag that turns on or off the progress bar. If false, then no thread is started or created.
```

Comment:
We are using numpy docstyle, [check out this portion](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) for how to specify parameters.